src/app/rmmapi/rmmhttpinterceptor.service.ts

```diff
@@ -63,6 +65,10 @@ export class RMMHttpInterceptorService implements HttpInterceptor {
                             req.url === '/ajax_mfa_authenticate' &&
                             r.body.is_2fa_enabled === '1') {
                                 console.log('proceed with 2fa login');
+                        } else if (
+                            req.url !== '/ajax_mfa_authenticate'
+                            && r.body.errors && r.body.errors.length) {
+                            this.snackBar.open(r.body.errors.join(' '), 'Dismiss');
```

Comment:
I'm not sure I'm a fan of this. There are already places in our code that catch request errors by themselves (and often display a snackbar when they find them, like here: https://github.com/runbox/runbox7/blob/master/src/app/contacts-app/contacts.service.ts#L66

Having a global HTTP interceptor catch all errors and pop them up on a screen may cause duplicate messages, or messages popping up in places where it doesn't make sense.

I think this should be handled by the caller of the request itself, not in the interceptor itself.